cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -343,13 +345,13 @@ def test_invalid_paging(self, page_number, page_size):
     def test_get_user_partitions_and_groups(self):
         self.course.user_partitions = [
             UserPartition(
-                id=0,
+                id=100,
```

Comment:
It actually could stay at 0 now, since the enrollment track partition is at 50. I was just trying to make the numbers more realistic, as they would have a minimum value of 100.

I don't want to put comments throughout the whole file, but I will put a place to put a general comment about IDs and the different ranges of numbers.